up.go

```diff
@@ -75,13 +75,32 @@ var (
 	unsafeMode   = pflag.Bool("unsafe-full-throttle", false, "enable mode in which command is executed immediately after any change")
 	outputScript = pflag.StringP("output-script", "o", "", "save the command to specified `file` if Ctrl-X is pressed (default: up<N>.sh)")
 	debugMode    = pflag.Bool("debug", false, "debug mode")
+	versionFlag = pflag.Bool("version", false, "display version and credits")
 )
 
 func main() {
 	// Handle command-line flags
 	pflag.Parse()
 
 	log.SetOutput(ioutil.Discard)
+	if *versionFlag {
+		os.Stdout.WriteString("up: Ultimate Plumber v" + version + " https://github.com/akavel/up\n")
+		os.Stdout.WriteString("up: \n")
+		os.Stdout.WriteString("up: Dependencies:\n")
+		os.Stdout.WriteString("up: \tTcell\n")
+		os.Stdout.WriteString("up: \t\thttps://github.com/gdamore/tcell\n")
+		os.Stdout.WriteString("up: \t\tLicense: Apache-2.0, ECL-2.0\n")
+		os.Stdout.WriteString("up: \n")
+		os.Stdout.WriteString("up: \tgo-isatty\n")
+		os.Stdout.WriteString("up: \t\thttps://github.com/mattn/go-isatty\n")
+		os.Stdout.WriteString("up: \t\tLicense: MIT, MIT-feh, JSON, Xnet\n")
+		os.Stdout.WriteString("up: \n")
+		os.Stdout.WriteString("up: \tpflag\n")
+		os.Stdout.WriteString("up: \t\thttps://github.com/spf13/pflag\n")
+		os.Stdout.WriteString("up: \t\tLicense: BSD-3-Clause, BSD-Source-Code, BSD-3-Clause-No-Nuclear-License-2014, BSD-4-Clause\n\n")
```

Comment:
WriteString is cool, thanks! Please follow up with the following changes:

 1. Put all the text in one multiline ["raw string"](https://golang.org/ref/spec#String_literals) (see below).
 2. Please move it to a ``const licenses = ...``, defined at the end of this file (see below).
 3. Please remove most of `up:` prefixes, keep only one on the first line (see below).
 4. Please also add "attribution" info for all the packages; for example, for tcell files, copy the [copyright line from the header, with authorship info](https://github.com/gdamore/tcell/blob/017915a4d77dabd7af10ab539e618a735d4b9c0a/attr.go#L1), like this:

        Copyright 2015 The TCell Authors

Points 1-3 together mean, that the final code should start roughly like this:

```go
const licenses = `up: Ultimate Plumber v` + version + ` https://github.com/akavel/up
Copyright 2018 the up AUTHORS
License: Apache-2.0  https://github.com/akavel/up/blob/804be3cfb8982f83e850f28bef5730672b6d04da/LICENSE

...
`